src/diffusers/models/attention.py

```diff
@@ -387,14 +389,17 @@ def __init__(
         activation_fn: str = "geglu",
         num_embeds_ada_norm: Optional[int] = None,
         attention_bias: bool = False,
+        do_self_attention: bool = True,
```

Comment:
just in terms of naming, the first attn is actually called "CrossAttention", so should we maybe just call it as follows:

```suggestion
        disable_first_cross_attention: bool = True,
```